app/src/main/java/org/simple/clinic/sync/DataSync.kt

```diff
@@ -14,11 +14,16 @@ class DataSync @Inject constructor(
     private val crashReporter: CrashReporter
 ) {
 
-  fun sync(): Completable {
-    return runAndSwallowErrors(modelSyncs.map { it.sync() })
+  fun sync(syncGroupId: SyncGroup?): Completable {
```

Comment:
I agree. I did not do a something like this because I was thinking of eliminating groups altogether in the future.
We could pass the `SyncConfig` instead.

Reply:
Cool. Let's revisit this at a later point in time then.